cli/src/main/scala/filodb.cli/CsvImportExport.scala

```diff
@@ -57,51 +54,15 @@ trait CsvImportExport {
   def ingestCSV(dataset: String, version: Int, csvPath: String, delimiter: Char) {
```

Comment:
Oh.  I think ScalaStyle should have caught this, but we might have turned off the style checker for parts of this file.